pgs_types.sql.in

```diff
@@ -132,6 +132,17 @@ CREATE FUNCTION spoly_out(spoly)
    LANGUAGE 'c'
    IMMUTABLE STRICT;
 
+CREATE FUNCTION spoly_deg(float8[])
+   RETURNS spoly
+   AS 'MODULE_PATHNAME', 'spherepoly_deg'
+   LANGUAGE 'c'
+   IMMUTABLE STRICT;
+
+COMMENT ON FUNCTION spoly_deg(float8[]) IS
```

Comment:
The comment here is different from the one in the upgrade script. They should be identical. I don't think including ` -- ` in front of the comment lines is appropriate with regards to how other `COMMENT ON FUNCTION`s are done elsewhere in pgSphere.